File: solutions/0190-reverse-bits/solution.java
public class Solution {
    // you need treat n as an unsigned value
    public int reverseBits(int n) {
        int ans=0,i=0;
        while(i<32)
        {
            ans<<=1;
            ans=ans|(n&1);
            n>>=1;
            i++;
        }
        return ans;
    }
}

File: solutions/0190-reverse-bits/solution.py
class Solution:
    # @param n, an integer
    # @return an integer
    def reverseBits(self, n):
        ans=0
        i=0;
        while(i<32):
            ans=ans<<1;
            ans=ans|(n&1);
            n>>=1;
            i+=1;
        return ans;
